frontend/app/src/components/wrapped/WrappedContainer.vue

```diff
@@ -257,7 +258,7 @@ defineExpose({
     </RuiAlert>
 
     <div class="flex flex-col md:flex-row md:grid-cols-4 gap-2 -mb-4 md:items-start">
-      <div class="mr-4 font-semibold">
+      <div class="mr-4 font-semibold my-2 whitespace-nowrap">
```

Comment:
I don't know where you do it but start ts should be start of data (this is how it was before this PR). Now it's 1/1/1970 (timestamp -> 0) which is not nice.